frontend/apps/reader/modules/readermenu.lua

```diff
@@ -175,7 +175,11 @@ function ReaderMenu:setUpdateItemTable()
             text = _("Screensaver"),
             sub_item_table = require("ui/elements/screensaver_menu"),
         }
-        table.remove(self.menu_items.screensaver.sub_item_table, 9)
+        local table_len = #self.menu_items.screensaver.sub_item_table
+        --- @todo Make this less prone to being messed up by screensaver menu changes, for example with MenuSorter.
+        if self.menu_items.screensaver.sub_item_table[table_len].text == ss_book_settings.text then
```

Comment:
I'll just write this line a bit more error-proof. Cf. https://github.com/koreader/koreader/issues/5469#issuecomment-541281013

Reply:
Still thinking ? Not going with your suggestion from https://github.com/koreader/koreader/issues/5469#issuecomment-541357260 ? (Which I guess you'll have to use at the 2 places you changed things in this PR.)